lighthouserc.cjs

```diff
@@ -1,3 +1,5 @@
+// LHCI currently does not support .cjs, so generate a json config with:
```

Comment:
Any reason for keeping this file if unsupported? Any issue with just keeping the json file?

Reply:
just that .cjs is easier to author, and it is on my backlog to update LHCI to support ESM config/at least support .cjs and then I'll come back and delete the JSON here.

I can do as you suggest if you'd prefer.